lib/main.dart

```diff
@@ -1,38 +1,37 @@
 import 'package:flutter/material.dart';
+import 'package:hacktoberfest_flutter/screens/home_page.dart';
 import 'package:provider/provider.dart';
 import 'package:hacktoberfest_flutter/providers/theme_provider.dart';
 import 'package:hacktoberfest_flutter/screens/splash_screen.dart';
+import 'package:animated_splash_screen/animated_splash_screen.dart';
 import 'routes.dart';
 
 void main() {
   runApp(const MyApp());
 }
 
 class MyApp extends StatelessWidget {
-  const MyApp({super.key});

```

Comment:
this is fine, its modern way of passing keys refer concept of parent classes in dart
and modify it back to this state